### PR TITLE
Tests!

### PR DESCRIPTION
**Pull request by connorsutton**
Thank you for contributing!
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] Your code doesn't break anything we can't fix
- [x] You have added appropriate tests

Please check one or more of the following to describe the nature of this PR:
- [x] New feature
- [x] Bug fix
- [ ] Documentation
- [ ] Other

### CODE REVIEW

File: .github/workflows/docker-hub.yml
Comment (joshuasundance-swca) [inline .github/workflows/docker-hub.yml:1]: undo

File: .github/workflows/hf-space.yml
Comment (joshuasundance-swca) [inline .github/workflows/hf-space.yml:1]: undo

File: README.md
Comment (joshuasundance-swca) [inline README.md:1]: undo

File: bumpver.toml
Comment (joshuasundance-swca) [inline bumpver.toml:1]: undo

File: docker-compose.yml
Comment (joshuasundance-swca) [inline docker-compose.yml:1]: undo change to image/service names

File: requirements.txt
Comment (joshuasundance-swca) [inline requirements.txt:1]: pin version to latest?

**Comment by joshuasundance-swca** [comment]
https://github.com/connorsutton/geospatial-data-converter/pull/1

**Comment by connorsutton** [comment]
Merged refactor and added topojson output option

**Comment by joshuasundance-swca** [comment]
https://github.com/connorsutton/geospatial-data-converter/pull/2

**Comment by joshuasundance-swca** [comment]
https://github.com/connorsutton/geospatial-data-converter/pull/3

**Pull request merged**